ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -107,14 +107,24 @@ objects:
 
         mkdir -p "${HOME}"
 
-        # wait until the setup job creates admin.kubeconfig
+        # wait until the upgrade job creates /tmp/shared/test
         while true; do
           if [[ ! -f /tmp/shared/test ]]; then
             sleep 15 & wait
             continue
           fi
           break
         done
+        echo "Found test flag"
+
+        # wait until the upgrade job creates admin.kubeconfig
+        while true; do
+          if [[ ! -f /tmp/shared/admin.kubeconfig ]]; then
+            sleep 15 & wait
+            continue
+          fi
+          break
+        done
```

Comment:
Is the other wait loop above still necessary if you add this?

Reply:
Right, its not necessary - test flag is created after admin.kubeconfig is being copied

Reply:
Fixed